integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/addsegment/AddSegmentTestCase.scala

```diff
@@ -542,7 +541,7 @@ class AddSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     copy(path.toString, newPath)
     checkAnswer(sql("select count(*) from addsegment1"), Seq(Row(30)))
 
-    sql(s"alter table addsegment1 add segment options('path'='$newPath', 'format'='parquet')").show()
+    sql(s"alter table addsegment1 add segment options('path'='$newPath', 'format'='PARQUET')").show()
```

Comment:
why is this change required?

Reply:
This change is required just for the testing purpose of uppercase format, so that I need not add a new testcase just for case sensitivity.